tokio/src/runtime/handle.rs

```diff
@@ -412,7 +416,7 @@ impl Handle {
         let future = super::task::trace::Trace::root(future);
         #[cfg(all(tokio_unstable, feature = "tracing"))]
         let future = crate::util::trace::task(future, "task", meta, id.as_u64());
-        self.inner.spawn_local(future, id, meta.spawned_at)
+        unsafe { self.inner.spawn_local(future, id, meta.spawned_at) }
```

Comment:
This function is very short, and it’s clear enough for developers to reason about the safety requirements, so we don’t need additional "Safety" comments here”